app/routes/verkiezingen/installatievergadering.js

```diff
@@ -45,29 +45,23 @@ export default class PrepareInstallatievergaderingRoute extends Route {
       params.bestuursperiode
     );
 
-    const ivStatuses = await this.store.findAll(
-      'installatievergadering-status'
-    );
-
-    const installatievergadering = (
-      await this.store.query('installatievergadering', {
-        'filter[bestuurseenheid][id]': bestuurseenheid.id,
-        'filter[bestuursperiode][:id:]': selectedPeriod.id,
-        include: ['status'].join(','),
-      })
-    )[0];
-
     const bestuursorganenInTijd =
       await this.getBestuursorganenInTijd(selectedPeriod);
 
+    await this.ivService.setup(selectedPeriod);
+    if (this.ivService.iv) {
+      await this.ivService.createBestuursorganenInTijdMap(
+        bestuursorganenInTijd
+      );
+    }
```

Comment:
Look where we use the isBehandeld and else just add the bestuursorganen in tijd to the setup method so we lose this if statement.

Reply:
removed it added the bois to the setup method